docker/app/Dockerfile

```diff
@@ -1,26 +1,84 @@
-FROM sillsdev/web-languageforge:app-base-staging
+ARG ENVIRONMENT=production
+
+# APP-BASE
+FROM php:7.3.28-apache AS base-app
+
+# install apt packages
+# p7zip-full - used by LF application for unzipping lexicon uploads
+# unzip - used by LF application for unzipping lexicon uploads
+# git - TODO: why do we need git?
+# gnupg2 - necessary for LFMerge package installation via SIL sources (will be uninstalled in production)
+# curl - used by LF application
+RUN apt-get update && apt-get -y install p7zip-full unzip git gnupg2 curl
+
+# see https://github.com/mlocati/docker-php-extension-installer
+# PHP extensions required by the LF application
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
+RUN install-php-extensions gd mongodb intl
```

Comment:
this might be the point where you would add a compound statement to remove anything that's not needed anymore.  We can chat about it if needed.